src/App.tsx

```diff
@@ -1,39 +1,81 @@
-import React from 'react';
+import { Component } from 'react';
 import './App.scss';
+import { ThemeProvider, createTheme } from '@mui/material/styles';
+import CssBaseline from '@mui/material/CssBaseline';
+import { Clock } from './Clock';
+
+const darkTheme = createTheme({
+  palette: {
+    mode: 'dark',
+  },
+});
+
+function App1() {
```

Comment:
move it to a separate file and use another name, avoid using numbers in the name